solr/core/src/java/org/apache/solr/api/ContainerPluginsRegistry.java

```diff
@@ -69,7 +72,8 @@
 public class ContainerPluginsRegistry implements ClusterPropertiesListener, MapWriter, Closeable {
   private static final Logger log = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
 
-  private final ObjectMapper mapper = SolrJacksonAnnotationInspector.createObjectMapper();
+  private static final ObjectMapper mapper = SolrJacksonAnnotationInspector.createObjectMapper()
+      .configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
```

Comment:
Is this a security risk? Why do we need this?

Reply:
This is not a security risk. In general it's better to ignore extra attributes for backcompat. Imagine adding an extra attribute to your plugin in a newer version and old Solr nodes failing because of that